README.md

```
@@ -91,25 +84,19 @@ Extract AI-generated suggestions from a specific PR.
-    }
+    "pr_number": 12345
   }
 }
 ```
 
 ### `pr_list`
-List all code suggestions from a PR with advanced filtering.
+List extracted PR review items from a previously processed PR.
 
 **Parameters:**
 - `owner` (string, required): Repository owner
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)